src/main/java/pong/Ball.java

```diff
@@ -1,4 +1,56 @@
 package pong;
 
 public class Ball {
+    private Paddle humanPaddle;
+    private Paddle compPaddle;
+
+    private double x;
+    private double y;
+    private double velX;
+    private double velY;
+    private final int MID_OF_BALL = PongView.BALL_WIDTH / 2;
+
+    public Ball(Paddle humanPaddle, Paddle compPaddle) {
+        //first position - middle of the board.
+        x = PongFrame.WIDTH / 2;
+        y = PongFrame.HEIGHT / 2;
+        velY = 1;
+        velX = -2;
```

Comment:
tbh, I did not understand why the guy in [this video](https://www.youtube.com/watch?v=xIqeK2hzx1I) decides to start with these values for velX and velY. I am trusting him on this one